src/main/java/org/vaadin/example/bookstore/ui/inventory/InventoryViewLogic.java

```diff
@@ -1,12 +1,12 @@
-package com.vaadin.samples.bookstore.ui.inventory;
+package org.vaadin.example.bookstore.ui.inventory;
 
 import java.io.Serializable;
 
```

Comment:
Add comments to setFragmentParameter explaining why we set the parameter; to keep the view state the same during e.g. a refresh and to enable bookmarking of individual product selections

Reply:
Done.